userguide/tutorials/invoice_examples.adoc

```diff
@@ -0,0 +1,132 @@
+= Invoice Examples
+
+This document provides examples of different invoice scenarios. 
+
+== Prerequisites
+
+Ensure that you have gone through the https://docs.killbill.io/latest/userguide_subscription.html#components-invoicing[_Subscription User Guide_] and are familiar with the basic invoicing concepts.
+
+
+== Invoice Terminology - work in progress
+
+Whenever there is a change in the system (a recurring subscription billing, an external charge, a change in plan, and so on), the system generates an invoice. An invoice consists of one or more invoice items of different types. In addition, an invoice consists of several components like `balance`, `chargedAmount`, `paidAmount` and so on. Let us first understand some of these terms. 
+
+
+* *Invoice Item*: Represents an amount corresponding to a type of charge, credit or adjustment. The different types of invoice items are `RECURRING`, `FIXED`, `EXTERNAL_CHARGE`, `USAGE`, `TAX`, `ITEM_ADJ`, `CREDIT_ADJ`, `REPAIR_ADJ` and `CBA_ADJ` as  explained in the https://docs.killbill.io/latest/userguide_subscription.html#components-invoicing[subscription billing] document. 
```

Comment:
An invoice item is more than an 'amount', i think could skip trying to explain what those are as this is not the purpose of the doc, and it's not so easy. If you can come with something good 👍 , if not skip the first sentence.


Also, I was looking at our [glossary](https://docs.killbill.io/latest/Kill-Bill-Glossary.html) and we don't see to have a definition for those (something to add in the future, maybe create a task to track it).

Reply:
Again, removed this section